ivy/functional/ivy/statistical.py

```diff
@@ -174,6 +174,219 @@ def mean(
            array ``x`` has an integer data type, the returned array must have the
            default floating-point data type.
 
+    Examples
+    --------
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.array([3., 4., 5.])
```

Comment:
Same comment here,

1. avoid repeating inputs;
2. add a one sentence description above

Reply:
Hi, here each example is denoted which kind of example it belongs to. I am just modifying the repeating inputs.